### DISCUSSION TITLE
How to make scroll a NewHBox ?

### DISCUSSION THREAD

**Discussion by lbellego** [fyne-io/fyne#3254] [Q&A] [answered]
I have a container with a vbox and another with a big label. On desktop it's displayed correctly. On mobile the result panel is on the first panel, merged on the first one.
How to make it scrolling ?

This is the last code from the function :
`return container.NewHBox(container.NewVBox(title, ligneSomme, ligneImpair, lignePremiers, ligneSous25, btnCalculer),
		x.resultatCalculs)`

**Comment by Jacalz** [comment, marked as answer]
I think you might want to change the HBox to something different. It will always compress items down to their horizontal MinSize.

A good option might be to use the `container.NewAdaptiveGrid(2, ...)`. This should make the result panel be on the right on desktop and then pop down below when on mobile.

**Comment by andydotxyz** [comment]
To more literally answer the question use `container.NewScroll` to wrap content in a scrollable container.
